content/templates/implementations.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Implementations
+date: "2024-01-4T12:01:05.284Z"
+description: "A curated list of awesome things related to Lottie"
+image: null
+---
+
+## Resources
+
+- <a href="https://lottiefiles.com/what-is-lottie" target="_blank">What is Lottie</a> by @lottiefiles
+- <a href="https://lottiefiles.github.io/lottie-docs/" target="_blank">Lottie Docs</a> by @lottiefiles
+- <a href="https://airbnb.io/lottie/#/" target="_blank">Lottie Docs</a> by @airbnb
+
+<br/>
+
+## Creation Tools
+
+<br/>
+
+##### Plugins
+
+- <a href="https://lottiefiles.com/ae" target="_blank">LottieFiles AE Plugin</a> for Adobe After Effects
+- <a href="https://aescripts.com/bodymovin/" target="_blank">Bodymovin</a> for Adobe After Effects
+
+##### For Design Tools
+
+- <a href="https://store.framer.com/package/airbnb/lottie" target="_blank">FramerX Lottie Package</a>
```

Comment:
Dead link, maybe should be https://www.framer.com/plugins/lottie ?

Reply:
Thanks! I cannot find the Airbnb's Framer for Lottie. The one you shared is LottieFiles's plugin - I will replace it instead of the dead link.